example/demo_catalog.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: "config.istio.io/v1alpha2"
```

Comment:
[istio/api](https://github.com/istio/api) configuration is versioned per component. Seems like the same versioning should be reflected in the CRD types, e.g.

```yaml
apiversion: broker.config.istio.io/v1alpha2   # => istio/api/broker/v1
apiVersion: proxy.config.istio.io/v1beta1     # => istio/api/proxy/v1
apiversion: mixer.config.istio.io/v1beta2     # => istio/api/mixer/v1
```